pkgs/by-name/cm/cmake/005-remove-systemconfiguration-dep.diff

```diff
@@ -1,7 +1,8 @@
-diff -Naur cmake-3.28.2/Utilities/cmcurl/CMakeLists.txt cmake-3.28.2-new/Utilities/cmcurl/CMakeLists.txt
---- cmake-3.28.2/Utilities/cmcurl/CMakeLists.txt	2024-01-29 23:01:32.000000000 +0300
-+++ cmake-3.28.2-new/Utilities/cmcurl/CMakeLists.txt	2024-02-16 13:09:40.805479195 +0300
-@@ -470,13 +470,6 @@
+diff --git a/Utilities/cmcurl/CMakeLists.txt b/Utilities/cmcurl/CMakeLists.txt
```

Comment:
How should we generate these patches?

Reply:
I just rebased the patches using `git rebase-patch`, part of `git-extras`